pyvrp/cpp/RandomNumberGenerator.h

```diff
@@ -88,6 +97,13 @@ RandomNumberGenerator::result_type RandomNumberGenerator::randint(T high)
     static_assert(std::is_integral<T>::value);
     return operator()() % high;
 }
+
+template <typename RandomIt>
+void RandomNumberGenerator::shuffle(RandomIt first, RandomIt last)
+{
+    for (auto idx = last - first - 1; idx > 0; --idx)
+        std::swap(first[idx], first[randint(idx + 1)]);
```

Comment:
Perhaps add a link to [this](https://en.cppreference.com/w/cpp/algorithm/random_shuffle.html#Version_2)?

Reply:
(or at least some backlink to `std::shuffle` on https://en.cppreference.com, because it's helpful to know where this came from for correctness, since we don't have test support here)